### PR TITLE
Add inz keyword support

### PR DESCRIPTION
**Pull request by parodi-smeup**
## Description

Include a summary of the change.

Related to # (issue)

## Checklist:
- [ ] There are tests regarding this feature
- [ ] The code follows the Kotlin conventions (run `./gradlew ktlintCheck`)
- [ ] The code passes all tests (run `./gradlew check`)
- [ ] There is a specific documentation in the `docs` directory

### CODE REVIEW

File: rpgJavaInterpreter-core/src/test/resources/QILEGEN/£JAX_D0.rpgle [hunk 1]
```diff
@@ -56,9 +56,9 @@
      D £JaxLLb                        5  0 INZ
      D £JaxLLc                        5  0 INZ
      D £JaxLBu                      140    INZ
-     D £JaxLSi                         Z   INZ
-     D £JaxLSt                         Z   INZ
-     D £JaxLEt                         Z   INZ
+     D***> £JaxLSi                         Z   INZ
```
Comment (lanarimarco) [inline rpgJavaInterpreter-core/src/test/resources/QILEGEN/£JAX_D0.rpgle:59]: Why?

**Review by lanarimarco — COMMENTED** [review]
I don't understand one thing.

File: rpgJavaInterpreter-core/src/test/resources/QILEGEN/£JAX_D0.rpgle (hunk 1 above)
Comment (parodi-smeup) [reply rpgJavaInterpreter-core/src/test/resources/QILEGEN/£JAX_D0.rpgle:59]: Due to conversation with Marco Benetti, we decided to temporarly remove fields defined as 'Z' type (timestamp).

**Review by lanarimarco — APPROVED** [review]
Ok

**Pull request merged**